src/test/java/mServer/testhelper/ZDFClientMock.java

```diff
@@ -23,8 +22,6 @@ public ZDFClient get() {
     }
     
     public void setUp(String url, String jsonFile) throws IOException, URISyntaxException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: java.io.IOException, java.net.URISyntaxException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1160)